.identity/99_main.tf.ci

```diff
@@ -1,30 +0,0 @@
-terraform {
-  required_version = ">=1.3.0"
-
-  required_providers {
-    azuread = {
-      source  = "hashicorp/azuread"
-      version = "2.30.0"
-    }
-    azurerm = {
-      source  = "hashicorp/azurerm"
-      version = "3.33.0"
-    }
-    github = {
-      source  = "integrations/github"
-      version = "5.12.0"
```

Comment:
why? this file is required into cicd workflow

Reply:
For what?

Reply:
ok we use sed to remove the backend remote state into CI/CD